eo-parser/src/test/java/org/eolang/parser/EoSyntaxTest.java

```diff
@@ -47,6 +47,7 @@
  *
  * @since 0.1
  */
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
@Yanich96 Is this necessary?

Reply:
@volodya-lombrozo Tests failed without it.